### PR TITLE
generating all possible subsequences

### PR DESCRIPTION
**Pull request by ujjawal-1999**
The assigned task, #410

### CODE REVIEW

**Comment by ujjawal-1999** [comment]
Kindly check and merge the code.

File: C-Plus-Plus/cp/Subsequences.cpp [hunk 1]
```diff
@@ -0,0 +1,86 @@
...
+	set<string>::iterator i;
+	for(it = m.begin(); it != m.end(); ++it){
+		for(i = it->second.begin(); i != it->second.end(); ++i)
+			cout << *i << " ";
+		cout << "\n";
+	}
+}
+void test() {
+    cout << "Test 1" << endl;
+    string a = "abc";
```
Comment (atarax665) [inline C-Plus-Plus/cp/Subsequences.cpp:74]: Don't hardcode inputs, take it from user.

**Review by atarax665 — CHANGES_REQUESTED** [review]

**Comment by atarax665** [comment]
@ujjawal-1999 We haven't heard from you in a while. Do the suggested changes and resolve the merge conflict ASAP.

**Review by atarax665 — COMMENTED** [review]
@ujjawal-1999 We haven't heard from you in a while. Do the suggested changes and resolve the merge conflict ASAP.

**Pull request closed without merging**